### PR TITLE
localize 404 page; allow Markdown at more places; clickable feature icons

### PR DESCRIPTION
**Pull request by salim-b**
I tried to translate the strings into the different supported languages as far as I could:

- `ca`, `de`, `en`, `fr`, `it`, `nl` are complete (thanks to [DeepL](https://deepl.com/translator) :grin:);
- in all the other languages only the `404NavHome` string is translated (took it from the `navHome` string), the other two strings `404Error` and `404Message` are still in English (waiting for someone more proficient to translate them);
- Markdown is supported in all the three 404 strings (processed by hugo's `markdownify`).

What I couldn't manage to translate is the page title. But as far as I understand, this [isn't supported by Hugo in any proper way yet](https://discourse.gohugo.io/t/404-change-title/158).

Besides I've

- added support for Markdown at various additional places (mostly (sub)titles defined in the site's config file);
- made the feature icons clickable by specifying the `link` key in the respective `data/features/[feature_name].yaml` files;
- replaced the Bootstrapious link in the footer with a more specific one pointing to the right template (universal) and using http**s**;
- made the `see_more` section a little more flexible by avoiding error(s) when `params.see_more.link_url` or `params.see_more.link_text` aren't defined.
- removed the duplicated site logo on the 404 page (identical to @timn's [commit](https://github.com/devcows/hugo-universal-theme/pull/137/commits/33ad328776eeb90f6d6793f7036fcb3338fb5155))

### CODE REVIEW

**Comment by ryanfox1985** [comment]
Hi @salim-b , thanks for the PR.

can you replace the `ca.yml`:
```
- id: 404Error
  translation: "Error 404: Pàgina no trobada"
 - id: 404Message
  translation: "Ho sentim - aquesta pàgina no es pot trobar en aquest lloc"
```

**Comment by salim-b** [comment]
> can you replace the `ca.yml`:
> 
> ```
> - id: 404Error
>   translation: "Error 404: Pàgina no trobada"
>  - id: 404Message
>   translation: "Ho sentim - aquesta pàgina no es pot trobar en aquest lloc"
> ```

Done.

Besides I've rename the `404GoHome` string to `404NavHome` to be more consistent with the existing terminology (`navHome` string).

**Comment by salim-b** [comment]
@ryanfox1985 what's holding you back to merge this pull request?

**Comment by ryanfox1985** [comment]
I want to test locally this PR.

**Comment by salim-b** [comment]
> I want to test locally this PR.

@ryanfox1985 Could you test it in the meantime?

File: layouts/partials/footer.html [hunk 1]
```diff
@@ -76,10 +76,10 @@ <h4>{{ i18n "contactTitle" }}</h4>
             <p class="pull-left">{{ .Site.Params.copyright | safeHTML }}</p>
             {{ end }}
             <p class="pull-right">
-              {{ i18n "templateBy" }} <a href="http://bootstrapious.com/free-templates">Bootstrapious</a>.
+              {{ i18n "templateBy" | markdownify }} <a href="https://bootstrapious.com/p/universal-business-e-commerce-template">Bootstrapious</a>.
```
Comment (ryanfox1985) [inline layouts/partials/footer.html]: nice!

File: exampleSite/data/features/consulting.yaml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 weight: 4
 name: "Consulting"
 icon: "fas fa-lightbulb"
+url: ""
```
Comment (ryanfox1985) [inline exampleSite/data/features/consulting.yaml]: @salim-b please can you add this parameter to the readme?  here https://github.com/salim-b/hugo-universal-theme/blob/e41d8712415a97e89bd667bf3afce75cf79c0a24/README.md#L283

after that I will merge the PR.

Thanks!

Comment (salim-b) [reply exampleSite/data/features/consulting.yaml]: Ok, done in https://github.com/devcows/hugo-universal-theme/pull/184/commits/d6c8db5c06dbb450a328b77b2d198800ea349f67

**Pull request merged**